backends/jgroups/src/test/resources/testing-flush-loopback.xml

```diff
@@ -27,14 +27,13 @@
 			xmit_table_num_rows="50"
 			xmit_table_msgs_per_row="1024"
 			xmit_table_max_compaction_time="30000"
-			max_msg_batch_size="100" />
+			discard_delivered_msgs="true" />
```

Comment:
I can't even begin to understand how this is related to our tests, so... I'll have to trust you :)